locales/sk/json.json

```diff
@@ -308,38 +308,38 @@
     "If you already have an account, you may accept this invitation by clicking the button below:": "Ak už máte účet, môžete túto pozvánku prijať kliknutím na tlačidlo nižšie:",
     "If you did not create an account, no further action is required.": "Ak ste si nevytvorili účet, nie je potrebná žiadna ďalšia akcia.",
-    "If you did not expect to receive an invitation to this team, you may discard this email.": "Ak ste neočakávali, že dostanete pozvánku do tohto tímu, môžete tento e-mail zahodiť.",
+    "If you did not expect to receive an invitation to this team, you may discard this email.": "Ak ste neočakávali, že dostanete pozvánku do tohto tímu, môžete tento email zahodiť.",
     "If you did not receive the email": "Ak ste email nedostali",
     "If you did not request a password reset, no further action is required.": "Ak ste nepožiadali o obnovenie hesla, tak považujte túto správu za bezpredmetnú a môžete ju vymazať.",
-    "If you do not have an account, you may create one by clicking the button below. After creating an account, you may click the invitation acceptance button in this email to accept the team invitation:": "Ak nemáte účet, môžete si ho vytvoriť kliknutím na tlačidlo nižšie. Po vytvorení účtu môžete kliknutím na tlačidlo prijatia pozvánky v tomto e-maile prijať pozvánku tímu:",
-    "If you need to add specific contact or tax information to your receipts, like your full business name, VAT identification number, or address of record, you may add it here.": "Ak potrebujete do svojich potvrdení pridať konkrétne kontaktné alebo daňové informácie, ako je celé vaše obchodné meno, IČ DPH alebo evidenčná adresa, môžete ich pridať sem.",
+    "If you do not have an account, you may create one by clicking the button below. After creating an account, you may click the invitation acceptance button in this email to accept the team invitation:": "Ak nemáte účet, môžete si ho vytvoriť kliknutím na tlačidlo nižšie. Po vytvorení účtu môžete kliknutím na tlačidlo prijatia pozvánky v tomto emaile prijať pozvánku tímu:",
+    "If you need to add specific contact or tax information to your receipts, like your full business name, VAT identification number, or address of record, you may add it here.": "Ak potrebujete do svojich potvrdení pridať konkrétne kontaktné alebo daňové informácie, ako je celé Vaše obchodné meno, IČ DPH alebo evidenčná adresa, môžete ich pridať sem.",
     "If you're having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Ak máte problém s kliknutim na tlačidlo \":actionText\", tak skopírujte a vložte nižšie uvedenú URL adresu do Vášho webového prehliadača:",
-    "Impersonate": "Zosobniť sa",
+    "Prihlásenie pod inou identitou": "Zosobniť sa",
```

Comment:
The names of the keys do not need to be translated.

Reply:
Changed